enterprise-suite/scripts/install-es.sh

```diff
@@ -41,6 +41,7 @@ function usage() {
     docvar ES_HELM_NAME "Helm release name"
     docvar ES_FORCE_INSTALL "Set to true to delete an existing install first, instead of upgrading"
     docvar DRY_RUN "Set to true to dry run the install script"
+    docvar ES_OPENSHIFT36 "set to true to use api versions for openShift 3.6"
```

Comment:
Are we likely to see other contexts with different versions down the road?  If so, I might prefer something like:
`ES_KUBERNETES=openshift36` (so we can later have something like `ES_KUBERNETES=minishift`)
(Or whatever is an appropriate name for `ES_KUBERNETES`.)

That way, when the time comes, we won't have to create a new env var or deal with conflicting settings.

Reply:
agree changed it ES_KUBERNETES